*: clarify rules and policies (#7823)
pingcap/docs: placement-rules-in-sql.md

>
> Placement Rules in SQL is an experimental feature introduced in v5.3.0. The syntax might change before its GA, and there might also be bugs. If you understand the risks, you can enable this experiment feature by executing `SET GLOBAL tidb_enable_alter_placement = 1;`.

> > **Note：**
> > The implementation of *Placement Rules in SQL* depends on *placement rules feature* of PD, refer [Configure Placement Rules](/configure-placement-rules.md). In the context of Placement Rules in SQL，*placement rules* may be referred to *placement policies* attached to other objects, or rules that sent from TiDB to PD.

Reviewer: ```suggestion
> **Note：**
>
> The implementation of *Placement Rules in SQL* relies on the *placement rules feature* of PD. For details, refer to [Configure Placement Rules](/configure-placement-rules.md). In the context of Placement Rules in SQL, *placement rules* might refer to *placement policies* attached to other objects, or to rules that are sent from TiDB to PD.
```

CREATE PLACEMENT POLICY myplacementpolicy PRIMARY_REGION="us-east-1" REGIONS="us-east-1,us-west-1";
```

> Then attach the policy to a table or partition using either `CREATE TABLE` or `ALTER TABLE`, we say it specified placement rules on the table or the partition:

Reviewer: ```suggestion
Then attach the policy to a table or partition using either `CREATE TABLE` or `ALTER TABLE`. Then, the placement rules are specified on the table or the partition:
```

| `SCHEDULE`                 | The strategy used to schedule the placement of followers. The value options are `EVEN` (default) or `MAJORITY_IN_PRIMARY`. |
| `FOLLOWERS`                | The number of followers. For example, `FOLLOWERS=2` means that there will be 3 replicas of the data (2 followers and 1 leader). |

> In addition to the placement options above, you can also use the advance configurations. For details, see [Advance placement option](#advanced-placement-option).

Reviewer: ```suggestion
In addition to the placement options above, you can also use the advance configurations. For details, see [Advance placement options](#advanced-placement-option)s.
```

> You can directly attach the default placement options to a database schema. This works similar to setting the default character set or collation for a schema. Your specified placement options apply when no other options are specified. For example:

Reviewer: Refer to https://github.com/pingcap/docs-cn/pull/8230/files#diff-5e2b3452de0170d2f05486307b1a3201e0ac80c785f89651f25fb53db5419b6aR151.
```suggestion
You can directly attach the default placement rules to a database schema. This works similar to setting the default character set or collation for a schema. Your specified placement options apply when no other options are specified. For example:
```

```sql
CREATE PLACEMENT POLICY p1 PRIMARY_REGION="us-east-1" REGIONS="us-east-1,us-east-2";  -- Create placement policies

CREATE PLACEMENT POLICY p2 FOLLOWERS=4;

CREATE PLACEMENT POLICY p3 FOLLOWERS=2;

CREATE TABLE t1 (a INT);  -- Creates a table t1 with no placement options.

ALTER DATABASE test POLICY=p2;  -- Changes the default placement option, and does not apply to the existing table t1.

CREATE TABLE t2 (a INT);  -- Creates a table t2 with the default placement policy p2.

> CREATE TABLE t3 (a INT) POLICY=p1;  -- Creates a table t3 without the default policy p2, because this statement has specified another placement policy.

Reviewer: refer to https://github.com/pingcap/docs-cn/pull/8230/files#diff-5e2b3452de0170d2f05486307b1a3201e0ac80c785f89651f25fb53db5419b6aR166
```suggestion
CREATE TABLE t3 (a INT) POLICY=p1;  -- Creates a table t3 without the default policy p2, because this statement has specified another placement rule.
```

ALTER DATABASE test POLICY=p3;  -- Changes the default policy, and does not apply to existing tables.

CREATE TABLE t4 (a INT);  -- Creates a table t4 with the default policy p3.

> ALTER PLACEMENT POLICY p3 FOLLOWERS=3; -- Tables with policy p3, i.e. t4, will have FOLLOWERS=3。

Reviewer: ```suggestion
ALTER PLACEMENT POLICY p3 FOLLOWERS=3; -- The table with policy p3 (t4) will have FOLLOWERS=3.
```

```

> One could use [ALTER PLACEMENT POLICY](/sql-statements/sql-statement-alter-placement-policy.md) to change a policy, and changes will be propagated to all objects with the corresponding policy.

Reviewer: ```suggestion
You can use [`ALTER PLACEMENT POLICY`](/sql-statements/sql-statement-alter-placement-policy.md) to change a policy, and the changes will propagate to all objects with the corresponding policy.
```

> ### Advanced placement option

Reviewer: ```suggestion
### Advanced placement options
```

> 

Reviewer: Line 214: placement policies -> placement rules, according to https://github.com/pingcap/docs-cn/pull/8230/files#diff-5e2b3452de0170d2f05486307b1a3201e0ac80c785f89651f25fb53db5419b6aR213

Line 216: placement options -> placement rules, according to  https://github.com/pingcap/docs-cn/pull/8230/files#diff-5e2b3452de0170d2f05486307b1a3201e0ac80c785f89651f25fb53db5419b6aR215

Author: What?
